lua/terrortown/menus/gamemode/appearance/miscellaneous.lua

```diff
@@ -6,5 +6,5 @@ CLGAMEMODESUBMENU.priority = 91
 CLGAMEMODESUBMENU.title = "submenu_appearance_miscellaneous_title"
 
 function CLGAMEMODESUBMENU:Populate(parent)
-
+    --local form = vgui.CreateTTT2Form(parent, "header_miscellaneous_settings")
```

Comment:
either keep it in there or remove it completely

Reply:
Guess it's better to have it at least there. I left it in and uncommented it.